### PR TITLE
_update_from_den_response bugfix

### PR DESCRIPTION
**Pull request by BelSasha**

### CODE REVIEW

**Comment by BelSasha** [comment]
* **#1441** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/1441?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/run-house/runhouse/1441?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: runhouse/resources/hardware/launcher_utils.py [hunk 1]
```diff
@@ -180,7 +180,7 @@ def _update_from_den_response(cls, cluster, config: dict):
             "ssh_properties",
             "client_port",
         ]:
-            value = config.get(attribute)
+            value = config.get("cluster_config").get(attribute)
```
Comment (jlewitt1) [inline runhouse/resources/hardware/launcher_utils.py:183]: we should prob leave this as is, and instead parse the response before calling `_update_from_den_response`. is the issue that the config being passed in has two keys (`cluster_config` and `launched_config`)? if so we can just set config=data["cluster_config"] in the `up` method

File: runhouse/resources/hardware/launcher_utils.py [hunk 2]
```diff
@@ -208,7 +208,9 @@ def up(cls, cluster, verbose: bool = True, force: bool = False):
                 payload=payload,
                 cluster_name=payload["cluster_config"].get("name"),
             )
-            cls._update_from_den_response(cluster=cluster, config=data)
+            cls._update_from_den_response(
```
Comment (jlewitt1) [inline runhouse/resources/hardware/launcher_utils.py:211]: actually don't we call this from two places? this addresses the streaming use case, but i imagine the same would apply if verbose=False?

Comment (BelSasha) [reply runhouse/resources/hardware/launcher_utils.py:211]: if verbose=False, the data dictionary we receive equals to the cluster_config. It is not a dict with two key. Is it expected or it is also might be related to the `is_up` issues?

**Comment by jlewitt1** [comment]
to be fixed in Den

**Pull request closed without merging**